test/test_manifest.json

```diff
@@ -5005,5 +5005,13 @@
       "rounds": 1,
       "type": "eq",
       "annotations": true
+    },
+    {
+      "id": "pr12828",
+      "file": "pdfs/pr12828.pdf",
+      "md5": "e44d364fba2f146aed04f9d9abbb0f28",
+      "rounds": 1,
+      "type": "eq",
+      "annotations": true
```

Comment:
Is this actually sufficient/correct, since another review comment mentioned using `"forms": true`?

Reply:
You're right.